verticapy/connection/connect.py

```diff
@@ -95,6 +115,31 @@ def set_connection(conn: Connection) -> None:
     ODBC connection. This should not be confused with a
     native   VerticaPy   connection  created   by   the
     'new_connection' function.
+
+    Example
+    -------
+    Create a connection using the official Vertica Python client:
+
+    .. note:: You can use any connector (ODBC, JDBC, etc.) as long it has both `fetchone` and `fetchall` methods. However, note that VerticaPy works most efficiently with the native client because of its support for various complex data types and certain Vertica optimizations.
+
+    .. code-block:: python
+
+        import vertica_python
+
+        conn_info = {'host': "10.211.55.14",
+                    'port': 5433,
+                    'user': "dbadmin",
+                    'password': "XxX",
+                    'database': "testdb"}
+        conn = vertica_python.connect(** conn_info)
+
+    Set up the connector:
+
+    .. code-block:: python
+
+        from verticapy.connection import set_connection
+
+        set_connection(conn)
```

Comment:
please add the warning:

.. warning:: This connector is used throughout the entire API. If it's closed, you'll need to create a new one. This is why, in some cases, it's better to use auto-connection. Auto-connection will automatically create a new connection if the current one is closed.